### DISCUSSION TITLE
Auto publish a submitted item?

### DISCUSSION THREAD

**Discussion by rsaunders2** [pimcore/pimcore#8630] [Q&A / Support] [answered]
I have set up a form to submit a blurb, in the form of a data object, from a registered user (not dissimilar to a comment section). It works near perfectly, except I'd like the submitted object to be published automatically. Would I need to set up a workflow, change certain permissions for said data object, or is there a function similar to save() that will allow me to publish the object. 

I apologize if this is very basic, and I simply missed it.

**Comment by passioneight** [comment, marked as answer]
You are probably looking for something like `$blurb->setPublished(true);` before calling `$blurb->save()`.

**Reply by rsaunders2** [reply]
Thanks a lot. I appreciate your quick response.
